src/EnergyPlus/FluidCoolers.cc

```diff
@@ -533,7 +473,7 @@ namespace FluidCoolers {
 				}
 			}
 
-			ErrorsFound = TestFluidCoolerTwoSpeedInputForDesign( cCurrentModuleObject, AlphArray, cNumericFieldNames, cAlphaFieldNames, FluidCoolerNum );
```

Comment:
This is odd.  In Github it shows this line went from:

```
ErrorsFound = TestFluidCoolerTwoSpeedInputForDesign( cCurrentModuleObje .....
```

to:

```
ErrorsFound = ErrorsFound || TestFluidCoolerTwoSpeedInputForDesign( cCurrentModuleObje .....
```

But then it looks like this routine is actually a new routine you extracted [below](https://github.com/NREL/EnergyPlus/pull/5305/files#diff-9065f7283e2d6dff1fa21c622d094aa6R506).  That's odd.  Makes it hard to see the diff.

Anyway, I parsed through it and see the small change [here](https://github.com/NREL/EnergyPlus/pull/5305/files#diff-9065f7283e2d6dff1fa21c622d094aa6R571).  Seems quite reasonable.  I'll run the defect file (assuming there is one) and verify and merge.